packages/near-fast-auth-signer/src/components/AddDevice/AddDevice.tsx

```diff
@@ -84,6 +84,7 @@ function AddDevicePage() {
   const defaultValues = {
```

Comment:
Bug: User login with e-mail on input instead of email used on passkeys


https://github.com/near/fast-auth-signer/assets/22358004/dfdacd95-6f4d-44c5-98d0-ba18d09734f3

Reply:
This will only happen if the email is pre filled. It's expected in this case because the user did not click Continue they clicked the input to edit the email

Reply:
But is this ok? The user select one passkey but endup on other account... I think it's a security problem, no? 

Imagine if 3 people are sharing the same computer and one can login into the other

Reply:
This is not possible unless the keypair for the selected account belongs to the first account

Reply:
Removed pre filling of email to address this